config/autoscaler.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: "autoscaling.openshift.io/v1"
+kind: "ClusterAutoscaler"
+metadata:
+  name: "default"
+spec:
+  podPriorityThreshold: -10
+  resourceLimits:
+    maxNodesTotal: 20
+    cores:
+      min: 8
+      max: 128
+    memory:
+      min: 4
+      max: 256
+  scaleDown:
+    enabled: true
+    delayAfterAdd: 10m
+    delayAfterDelete: 5m
+    delayAfterFailure: 30s
+    unneededTime: 30m
+---
+apiVersion: "autoscaling.openshift.io/v1beta1"
+kind: "MachineAutoscaler"
+metadata:
+  name: "worker-us-east-1c"
+  namespace: "openshift-machine-api"
+spec:
+  minReplicas: 0
```

```suggestion
  minReplicas: 1
```
according to the docs, we should not set this value to `0` https://docs.openshift.com/container-platform/4.2/machine_management/applying-autoscaling.html#machine-autoscaler-cr_applying-autoscaling

---

Yeah.. not sure why though. It seems to be working fine.
Anyway I switched to 1b machine set since we currently don't have any machines in 1c and I don't want to scale it up: https://github.com/codeready-toolchain/toolchain-infra/pull/8/commits/ec4141920a65d92d4b92c40693ac6306668e0627